examples/monaco.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <!-- Firebase -->
+    <script src="https://www.gstatic.com/firebasejs/3.3.0/firebase.js"></script>
+
+    <!-- Firepad -->
+    <link rel="stylesheet" href="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.css" />
+    <script src="https://cdn.firebase.com/libs/firepad/1.4.0/firepad.min.js"></script>
+
+    <!-- Loader.js -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.13.1/min/vs/loader.js"></script>
+
+    <style>
+        html { height: 100%; }
+        body { margin: 0; height: 100%; position: relative; }
+        /* Height / width / positioning can be customized for your use case.
+           For demo purposes, we make firepad fill the entire browser. */
+        #firepad {
+            width: 100%;
+            height: 100%;
+        }
+    </style>
+
+</head>
+<body onload="init()">
+<div id="firepad"></div>
+
+<script>
+    function init() {
+        //// Initialize Firebase.
+        //// TODO: replace with your Firebase project configuration.
+        var config = {
+            apiKey: "AIzaSyC_JdByNm-E1CAJUkePsr-YJZl7W77oL3g",
+            authDomain: "firepad-tests.firebaseapp.com",
+            databaseURL: "https://firepad-tests.firebaseio.com"
+        };
+        firebase.initializeApp(config);
+
+        //// Get Firebase Database reference.
+        var firepadRef = getExampleRef();
+
+        //// Create Monaco and firepad.
+        require.config({ paths: {'vs': './../node_modules/monaco-editor/min/vs'}});
```

Comment:
Hrm...  You're pulling in loader.js from the CDN but then loading from node_modules here...  Ideally we'd use one or the other (and the CDN would be best so people can just copy/paste the example and have it work).

Reply:
I'm not that sure about the CDN thing, so added loader import from local only.